### ISSUE TITLE
"error: unknown attribute" on doc comments

### ISSUE THREAD

**Issue by bjorn3** [Rust-GCC/gccrs#1125]
I tried this code:

```rust
/// An implementation of SipHash 1-3.
///
/// This is currently the default hashing function used by standard library
/// (e.g., `collections::HashMap` uses it by default).
///
/// See: <https://131002.net/siphash>
pub struct SipHasher13 {}
```

I expected to see this happen: Compiles fine 

Instead, this happened:

```
<source>:151:1: error: unknown attribute
  151 | /// An implementation of SipHash 1-3.
      | ^
```

### Meta

- What version of Rust GCC were you using, git sha if possible: gccrs (Compiler-Explorer-Build-gcc-60324125c3dbfd2f1551ec41dcfd4322ca80d6fa-binutils-2.36.1) 12.0.1 20220118 (experimental)

**Comment by bjorn3** [comment]
Also `#[doc(hidden)]` is not supported.

**Comment by liushuyu** [comment]
Fix proposed in #1139

**Comment by liushuyu** [comment]
This is already fixed.
